web/modules/custom/joinup_core/js/sign_in_destination.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file
+ * Attaches a destination parameter to the sign in link to the current page.
+ */
+
+(function ($, Drupal, drupalSettings) {
+
+  /**
+   * Alters the sign in link to have a destination parameter to the source page.
+   *
+   * The home page 'Sign in' link and the 'Sign in' page 'Sign in' link are not
```

Comment:
This text is a bit weird, and it's written down in the code already, so it can be removed.